ctest-next/src/translator.rs

```diff
@@ -26,24 +27,29 @@ impl TranslationError {
         Self {
             kind,
             source: source.to_string(),
-            span,
+            span: format!(
+                "line {}, column {}, in {}",
```

Comment:
It's kind of standard to do `foo.rs:10:20`, some terminals will hyperlink that for you

```rust
"{fname}:{line}:{col}",
// FIXME(ctest): ...
fname = "<unknown file>"
line = span.start().line,
col = span.start().column,
```